cms/migrations/0002_auto_20200505_0933.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.10 on 2020-05-05 09:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.blocks
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0045_assign_unlock_grouppagepermission'),
+        ('wagtailforms', '0004_add_verbose_name_plural'),
+        ('klasses', '0014_no_max_len_application_stage'),
+        ('wagtailimages', '0001_squashed_0021'),
+        ('wagtailredirects', '0006_redirect_increase_max_length'),
+        ('cms', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BootCampRunPage',
+            fields=[
+                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+                ('description', wagtail.core.fields.RichTextField(blank=True, help_text='The description shown on the product page')),
+                ('content', wagtail.core.fields.StreamField([('rich_text', wagtail.core.blocks.RichTextBlock())], blank=True, help_text='The content of the benefits page')),
+                ('klasses', models.OneToOneField(help_text='The Klass for this page', null=True, on_delete=django.db.models.deletion.SET_NULL, to='klasses.Klass')),
```

Comment:
Are you going to wait for @gsidebo PR for Klasses rename?

Reply:
yes